v1/format/format.go

```diff
@@ -927,7 +1102,10 @@ func (w *writer) writeRef(x ast.Ref, comments []*ast.Comment) []*ast.Comment {
 				w.writeBracketed(w.formatVar(p))
 			default:
 				w.write("[")
-				comments = w.writeTerm(t, comments)
+				comments, err = w.writeTerm(t, comments)
+				if err != nil {
+					w.write("\n")
```

Comment:
For the uninitiated, it's hard to grasp why we'd want to add a newline here on error. A comment would be helpful. A typed error might be even more helpful.

Reply:
Definitely a harsh initiating trial messing with `format.go` haha added a custom error check and a comment to ease future suffering.